src/components/landing/pages/intro/cover-buttons/cover-buttons.tsx

```diff
@@ -1,17 +1,16 @@
 import React from 'react'
-import { Link } from 'react-router-dom'
 import { Button } from 'react-bootstrap'
-import { LoginStatus } from '../../../../../redux/user/types'
 import { Trans, useTranslation } from 'react-i18next'
 import { useSelector } from 'react-redux'
+import { Link } from 'react-router-dom'
 import { ApplicationState } from '../../../../../redux'
 import './cover-buttons.scss'
 
 export const CoverButtons: React.FC = () => {
   useTranslation()
   const user = useSelector((state: ApplicationState) => state.user)
 
-  if (user.status === LoginStatus.ok) {
+  if (!user) {
```

Comment:
The logic is inverted here
```suggestion
  if (user) {
```

Reply:
good point